scripts/setup/bootstrap.sh

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
Did this just get moved with changes to  `s|scripts/|scripts/setup/|` and git's move detection failed somehow?  If not, what are the other changes here?

Reply:
Yes it was moved to `/scripts/setup/`, but in order to maintain backward compatibility there is a symlink in `/scripts/`, which confuses git about rename... I would like to make it a rename (so the history will be preserver), but I don't know how to do this in a single PR... Normally, I would rename in one commit, and add symlink in the other.